docs/spec/software/abci.md

```diff
@@ -52,14 +52,13 @@ objects in the `ResponseBeginBlock`:
 
 ```
 message Validator {
```

Comment:
`ValidatorUpdate`. also need to add `Validator (address, power)` struct

Reply:
This is just some convenience notes that we might be able to remove completely now that ABCI has been merged into Tendermint. Note it should all be pretty redundant with what's in the actual ABCI spec, but it's also replicated here for now. 

We could add notes on the BeginBlock stuff here, but not sure it's worth it since we probably want to deprecate this file in favour of the actual ABCI spec. 

Thoughts?

Reply:
👍 for deprecating this stuff